README.md

```diff
@@ -48,8 +48,25 @@ not critical to users, and left out.
 - hoppers, dispensers, droppers
 - entities
 
+### Extra Notes
 
-License:
+Now allows a choice of conversion to Minetest Game or MineClone2.
+
+Currrently compatible with Minecraft Worlds up to, & including v1.12.2
+
+
+##### Coordinate Conversion from Minecraft to Minetest:
+```
+X = Approximately the same in both.
+Y = Approximately -64 nodes.
+Z = The number is approximately inverted.
+
+Example of converted coordinates:
+Minecraft: 704, 4, -318
+Minetest: 720, -60, 334
```

Comment:
should we maybe include a (lua) example function that converts the old coordinates?

Reply:
Possibly? I'm not sure how that'd work. I just know that converted maps don't keep the player's save location; so spawning into an imported map requires flying/teleportation to the converted area. And, it's quickest if users know the coordinates beforehand so they have a rough idea of where to look.